src/screens/Search/Lists/ResultsList.js

```diff
@@ -47,8 +47,8 @@ const renderItem = ({ item }, imageSize, onPress) => {
     <TouchableOpacity onPress={() => onPress(item)}>
       <View
         style={{
-          height: h - (m * 2),
-          width: w - (m * 2),
+          height: h - m * 2,
```

Comment:
I think this breaks an eslint rule about mixed operators - you're likely going to need the `( )` -- I know this isn't a new file but if you wouldn't mind also changing this file to use better variables name that would be :1st_place_medal: